dev/dev_setup.sh

```diff
@@ -279,13 +287,20 @@ put_brew_in_path() {
 set_ubuntu_language_encoding() {
     case $CRYPTOL_PLATFORM in
         $UBUNTU20 | $UBUNTU22)
-            notice "Adding language environment variables to $VAR_FILE"
-            echo "export LANG=C.UTF-8" >> $VAR_FILE
-            echo "export LC_ALL=C.UTF-8" >> $VAR_FILE;;
-        *) ;;
+            if $LANG!= *"UTF-8"* || $LANG != *"utf-8"*; then
+                notice "Language environment variables are not set as expected."
+                notice "    You may need to set them to UTF-8."
+                notice "    Uncomment the LANG var in $VAR_FILE before sourcing to do so"
+                echo "# uncomment the following lines to fix decoding errors e.g. hGetContents" >> $VAR_FILE
+                echo "# export LANG=C.UTF-8" >> $VAR_FILE
```

Comment:
I remember some time back I filed a bug report with I think Debian that there was no C.UTF-8 locale and they rejected it. That was years ago and they have probably grown up since, but maybe not.  (No need to do anything different here for now, but we might end up needing to revisit it sometime later. JFYI)